lib/src/intl_suggestors/intl_migrator.dart

```diff
@@ -81,6 +81,9 @@ class IntlMigrator extends ComponentUsageMigrator {
 
   IntlMigrator(this._className, this._messages);
 
+  String get ignoreStatement => 'IGNORE STATEMENT - INTL';
+  String get ignoreFile => 'IGNORE FILE - INTL';
```

Comment:
```suggestion
  String get ignoreFile => 'ignore_for_file: intl';
```

Reply:
I'm not sure we should use exactly the same comment as the linter. That might confuse people who would think it's a lint rule, not a directive.

It looks like there are `orcm_ignore` and `orcm_ignore_for_file` comments in the codemod already (see README.md, lib/src/util/ignore_info.dart) but it's not a perfect fit. But maybe we should have a parallel intl_ignore:, or intl_codemod_ignore:?

Reply:
changed these to:  `//ignore_statement: intl_message_migration` and `//ignore_file: intl_message_migration` respectively.